packages/purpose-process/src/resources/templates/documents/riskAnalysisTemplate.html

```diff
@@ -48,12 +56,24 @@ <h1>Analisi del rischio</h1>
       </div>
 
       <div class="item">
-        <div class="label">Erogatore</div>
+        <div class="label">Ente erogatore</div>
         <div class="value">{{producerText}}</div>
```

Comment:
Replace `producerText` with the same approach used for producer delegate: name + IpaCode if present